packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -364,6 +365,15 @@ export class Viewer {
    */
   @Prop({ mutable: true }) public pmi: PmiController | undefined;
 
+  /**
+   * The controller for accessing and viewing SceneItems.
+   *
+   * @readonly
+   */
+  @Prop({ mutable: true }) public sceneItemController:
```

Comment:
How would you feel about renaming this to `sceneItems` and omitting the `Controller` part to follow our other controllers exposed this way? (thining about `pmi` and `modelViews`)